apps/prairielearn/src/pages/authLogin/authLogin.html.ts

```diff
@@ -114,17 +114,18 @@ export function AuthLogin({ institutionAuthnProviders, service, resLocals }: Aut
           <div class="login-container">
             <div>
               <h1 class="text-center">PrairieLearn</h1>
-              <h2 class="text-center subheader">
+              <h2 class="text-center subheader mb-5">
                 Sign in ${service ? `to continue to ${service}` : ''}
               </h2>
-              <div class="login-methods mt-5">
-                ${resLocals.devMode
-                  ? html`
-                      <a class="btn btn-success w-100" href="/pl/dev_login" role="button">
-                        <span class="font-weight-bold">DevMode by-pass</span>
-                      </a>
-                    `
-                  : null}
+              ${resLocals.devMode
+                ? html`
+                    ${DevModeBypass()}
+                    <hr />
+                    ${DevModeLogin({ csrfToken: resLocals.__csrf_token })}
```

Comment:
One minor issue here was that I can open this login page without setting `authType`, but the login won't do anything. While that's understandable, it can be confusing. I wonder if it may be better to hide this form if `authType` is `none`, or at least show a message that config needs to be updated?

Reply:
That seems reasonable! I'll make that change.

Reply:
#8205 removed `authType` entirely 🎉